### PR TITLE
Redirect from One Path or Multiple Paths

### LINKED ISSUE #3531: See More Redirect Scenarios - Old New
Re: [Domains and Redirects](https://pantheon.io/docs/domains/)

Priority (Medium):

## Issue Description:
The **Redirect From One Path to Another** section seems to be not working and at times causing a redirect loop

## Suggested Resolution
Provide a new set of redirects:

For single paths:
```
// 301 Redirect from /old to /new
// Check if Drupal or WordPress is running via command line
    if (($_SERVER['REQUEST_URI'] == '/old') && (php_sapi_name() != "cli")) {
        header('HTTP/1.0 301 Moved Permanently');
        header('Location: https://'. $_SERVER['HTTP_HOST'] . '/new');
        exit();
    }
```

For multiple paths and redirects
```
$redirects = array(
    "/", 
    "/old", 
    "/another/path", 
    "/old-path");

// 301 Redirect from multiple paths
// Check if Drupal or WordPress is running via command line
    if ((in_array($_SERVER['REQUEST_URI'], $redirects)) && (php_sapi_name() != "cli")) {
	header('HTTP/1.0 301 Moved Permanently');
	header('Location: https://'. $_SERVER['HTTP_HOST'] . '/new-path-for-all');
	exit();
    }
```

### PR DESCRIPTION
**Pull request by eabquina**
Closes #3531

## Effect
PR includes the following changes:
- A slight modification of the code snippet
- Additional example from Path Redirects
-

## Remaining Work
- [ ] List any outstanding todos
- [ ] If needed

## Post Launch
- [ ] Redirect `/docs/old-path/` => `/docs/new-path/`
- [ ] Include/exclude pages ^ respectively within docs search service provider

### CODE REVIEW

**Comment by thomas-thackery** [comment]
👍

**Comment by alexfornuto** [comment]
Thanks @thomas-thackery for reviewing!

**Comment by alexfornuto** [comment]
Tested and working! I'm going to merge, but @rachelwhitton feel free to review the copy if you like.

File: source/_docs/domains.md [hunk 1]
```diff
@@ -139,16 +139,36 @@ The same technique works for single subdomain redirects. Just specify the path i
 
 #### Redirect From One Path to Another
 
+For a single path:
```
Comment (rachelwhitton) [inline source/_docs/domains.md]: @alexfornuto can you add stronger intros to help contextualize the usage of the snippet?

**Pull request merged**